CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 see also the changelogs of `smtlib-backends-tests`, `smtlib-backends-process` and 
 `smtlib-backends-z3`
 
+# v0.3-alpha
```

Comment:
I used semantic versioning here to indicate that this isn't supposed to be released as is. I'll just keep adding changes under `v0.3-alpha` until we want to create a new release, at which point I'll just rename it `v0.3-alpha`. I was thinking of using the hackage scheme otherwise, e.g. if the changes hadn't been breaking I could have used `v2.1-alpha` until the next release. tl;dr: the "alpha" is just some internal tag that won't appear in releases.

Does that sound good ?